frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -0,0 +1,119 @@
+                <Explain :text="debtDai.toFixed()"></Explain> =
+                <FormatCurrency :value="collateralDifference" currency="DAI"
+            /></span>
+            <div v-else>
+                <span class="opacity-50">Unknown</span>
+                <span>DAI</span>
+            </div>
+        </div>
+        <div class="flex justify-end mt-4">
+            <BaseButton
+                type="primary"
+                :disabled="isRefreshing"
+                :is-loading="isRefreshing"
+                @click="$emit('refreshLimits')"
+            >
+                Refresh
+            </BaseButton>
+        </div>
+    </BasePanel>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import BigNumber from 'bignumber.js';
+import BaseButton from '~/components/common/inputs/BaseButton.vue';
+import TextBlock from '~/components/common/other/TextBlock.vue';
+import Explain from '~/components/common/other/Explain.vue';
+import FormatCurrency from '~/components/common/formatters/FormatCurrency.vue';
+import BasePanel from '~/components/common/other/BasePanel.vue';
+import { LiquidationLimits } from '~/../core/src/types';
+
+export default Vue.extend({
+    name: 'VaultLiquidationLimitsCheckPanel',
+    components: {
+        BaseButton,
+        BasePanel,
+        TextBlock,
+        Explain,
+        FormatCurrency,
+    },
+    props: {
+        liquidationLimits: {
+            type: Object as Vue.PropType<LiquidationLimits>,
+            required: true,
+        },
+        debtDai: {
+            type: BigNumber,
+            required: true,
+        },
+        collateralType: {
+            type: String,
+            required: true,
+        },
+        isRefreshing: {
+            type: Boolean,
+            default: false,
+        },
+        isExplanationsShown: {
+            type: Boolean,
+            default: true,
+        },
+    },
+    computed: {
+        globalDifference(): BigNumber {
+            return this.liquidationLimits.maximumProtocolDebtDai
+                .minus(this.liquidationLimits.currentProtocolDebtDai)
+                .minus(this.debtDai);
+        },
+        collateralDifference(): BigNumber {
+            return this.liquidationLimits.maximumCollateralDebtDai
+                .minus(this.liquidationLimits.currentCollateralDebtDai)
+                .minus(this.debtDai);
+        },
+        isGlobalLimitsReached(): Boolean {
+            if (this.globalDifference.isNegative()) {
+                return true;
+            }
+            if (this.collateralDifference.isNegative()) {
+                return true;
+            }
+            return false;
+        },
+        currentStateAndTitle(): PanelProps {
+            if (this.isGlobalLimitsReached) {
+                return {
+                    name: 'incorrect',
+                    title: 'Current global liquidation limits are reached',
```

Comment:
@valiafetisov do we want to remove the `global` in the panel messages as we have:

1.) Global limits
2.) Collateral limits

I think this would make it easier to understand.

Reply:
I think the message should be different in case collateral or global limits were reached. In the mock it is just the example for one case

Reply:
Agreed! However I would add something after the "please wait". Right now I would be wondering, please wait for what?

My proposal therefore: "..., please wait for them to lower"

Reply:
(you can give a priority for the global limit in case both are reached to not handle the `&&` edge case)

Reply:
Then let's move the whole `please wait` part to the explanation of this panel